src/testFixtures/workspaceFolder/python3.7-image-sam-app/hello_world/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 628053151772.dkr.ecr.sa-east-1.amazonaws.com/awslambda/python3.7-runtime:beta
+FROM public.ecr.aws/lambda/python:3.7
```

Comment:
(comment not specific to this file)

Do we need to cover the other runtimes? Why are only three represented in this PR?